evap/staff/views.py

```diff
@@ -1598,6 +1598,8 @@ def notify_reward_points(grantings, **_kwargs):  # pylint: disable=unused-variab
         form.save()
         delete_navbar_cache_for_users([user])
         messages.success(request, _("Successfully updated user."))
+        for message in form.remove_messages:
+            messages.warning(request, message)
```

Comment:
codecov reports line 1602 not being covered by tests. did you miss it or was including it in a test too complex?

Reply:
didnt notice because it broke when adding "mocking" to the test. Got fixed in the last commit